dockstore_launcher_config/compose.config

```diff
@@ -29,5 +29,5 @@
 "GITHUB_APP_NAME":"foobar",
 "DOCUMENTATION_URL":"foobar",
 "FEATURED_CONTENT_URL":"foobar",
-"GALAXY_PLUGIN_VERSION":"NONE"
+"GALAXY_PLUGIN_VERSION":""
```

Comment:
Was more thinking `null` default would be `"GALAXY_PLUGIN_VERSION": null`
https://stackoverflow.com/questions/21120999/representing-null-in-json#21121267
I don't feel strongly about it though